test/spec.js

```diff
@@ -143,6 +143,60 @@ test('operation timeout before attempt timeout', async t => {
     }
 });
 
+test('dread.exp no jitter', t => {
+    let exp = dread.exp({
+		factor: Number(1),
+		jitter: Boolean(false)
+	});
+
+	t.is(exp(1), 100);
+	t.is(exp(2), 100);
+	t.is(exp(3), 100);
+
+	exp = dread.exp({
+		factor: Number(2),
+		jitter: Boolean(false)
+	});
+	t.is(exp(1), 200);
+	t.is(exp(2), 400);
+	t.is(exp(3), 800);
+
+   exp = dread.exp({
+		factor: Number(3),
+		jitter: Boolean(false)
+	});
+	t.is(exp(1), 300);
+	t.is(exp(2), 900);
+	t.is(exp(3), 2700);
+});
+
+test('dread.exp capped at limit', t => {
+    const exp = dread.exp({
+		factor: Number(3),
```

Comment:
no need to box everything with Number/Boolean, 3 is a Number, false is a Boolean